.bazelrc

```diff
@@ -121,6 +121,10 @@ build:qemu-bpf --sandbox_fake_username
 build:qemu-bpf --//bazel/cc_toolchains:libc_version=glibc2_36
 build:qemu-bpf --//bazel/test_runners:test_runner=qemu_with_kernel
 build:qemu-bpf --run_under="bazel/test_runners/qemu_with_kernel/test_runner.sh"
+# TODO(ddelnano): Temporary until https://github.com/pixie-io/pixie/issues/1347
+# is addressed. This is to limit the size of the change needed to fix
+build:qemu-bpf --define PL_BPF_TESTING=true
```

Comment:
It seems like `PL_BPF_TESTING` isn't used in any bazel build files so the `copt` should be sufficient.

Reply:
I cargo culted this from an existing example, but I see now that it isn't necessary. Updated in the latest revision.